rest/blip_sync.go

```diff
@@ -300,10 +300,15 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, params *subChangesParams
 		}
 	}()
 
+	// Don't send conflicting rev tree branches, just send the winning rev tree branch leaf revision + history.
+	// Otherwise, Couchbase Lite will non-deterministically choose an arbitrary branch as the winner and
+	// tombstone the other branch.  See LiteCore #437.
```

Comment:
I don't think the non-deterministic branch choosing is really the reason we want to set sendConflicts=false.  It's more correct to say here that CBL 2.0 (and blip_sync) doesn't support branched revision trees, and so we only want to send the winning revision.

Reply:
I guess the angle I was taking was to try to scare anyone who decided to flip that to true on a whim to fix some other bug.  At least, let them know the consequences.  But yeah, I agree that's not the primary motivation, so I'll reword that and leave the link to the issue for further backstory.